pkg/graph/resolvers/system_intake.go

```diff
@@ -113,6 +114,13 @@ func SystemIntakeUpdate(ctx context.Context, store *storage.Store, fetchCedarSys
 	intake.EASupportRequest = null.BoolFromPtr(input.NeedsEaSupport)
 	intake.HasUIChanges = null.BoolFromPtr(input.HasUIChanges)
 	intake.UsesAITech = null.BoolFromPtr(input.UsesAiTech)
+	intake.UsingSoftware = null.StringFromPtr(input.UsingSoftware)
+
+	// Create string array from SoftwareAcqisitionMethods enum array
+	acqMethods := lo.Map(input.AcquisitionMethods, func(acqMethod models.SystemIntakeSoftwareAcquisitionMethods, idx int) string {
+		return acqMethod.String()
+	})
+	intake.AcquisitionMethods = acqMethods
```

Comment:
nit - can skip temp var

```suggestion
	intake.AcquisitionMethods = lo.Map(input.AcquisitionMethods, func(acqMethod models.SystemIntakeSoftwareAcquisitionMethods, idx int) string {
		return acqMethod.String()
	})
```

Reply:
good call meant to refactor this at some point but lost track of it. fixed.